addons/website_event_meet/views/event_meet_templates_list.xml

```diff
@@ -179,8 +179,8 @@
                 <button disabled="disabled" class="btn btn-primary align-self-start">Create a Room</button>
                 Room creation will be available when event starts at 
                 <span>
-                    <span class="fw-bold" t-field="event.with_context(tz=event.date_tz).date_begin"
-                        t-options="{'format': 'medium'}"/>
+                    <span class="fw-bold" t-field="event.date_begin"
+                        t-options="{'format': 'medium', 'tz_name': event.date_tz}"/>
```

Comment:
Shouldn't we actually use ``date_begin_located`` ? Did you try with playing withous variou tz, as the purpose is to correctly display the TZ when event and customer's TZ are different ?

Reply:
`date_begin_located` (char field) could be used because here they are the same options. However, by doing this no widget options can be chosen and editing can no longer be done via the website (I have not looked at the whole feature but as it is for website I think that will be editable).